### ISSUE TITLE
some effect types gives error while trying to register. [only on localhosted server]

### ISSUE THREAD

**Issue by FlameDarkk** [Sashie/skDragonRecode#6]
while `register new particle effect sphere with id "test1"` is working fine if I change the "sphere" part to "circle", it gives the following error ;
```
[13:36:42 INFO]:     Can't understand this condition/effect: register new particle effect circle with id "uniqueID"
[13:36:42 INFO]:     Line: register new particle effect circle with id "uniqueID"
```
Tested on 1.20.4 latest papermc and latest skript.
The full line that gives error `register new particle effect circle with id "asd1"` which is from the example skript provided from Sashie

Sk Info :
[13:42:07 INFO]: [Skript] Server Version: git-Paper-435 (MC: 1.20.4)
[13:42:07 INFO]: [Skript] Skript Version: 2.8.2 (skriptlang-github)
[13:42:07 INFO]: [Skript] Installed Skript Addons:
[13:42:07 INFO]: [Skript]  - SkDragonRecode v1.0.2
[13:42:07 INFO]: [Skript]  - Skacket v1.1.0
[13:42:07 INFO]: [Skript]  - PoaSK v3.2
[13:42:07 INFO]: [Skript]  - skript-gui v1.3 (https://github.com/APickledWalrus/skript-gui)
[13:42:07 INFO]: [Skript]  - SkGlow v2.0.1 (https://github.com/MinecraftMan1013number2/skGlow)
[13:42:07 INFO]: [Skript]  - skRayFall v1.9.28 (https://sk.rayfall.net/)
[13:42:07 INFO]: [Skript]  - SkBee v3.3.0 (https://github.com/ShaneBeee/SkBee)
[13:42:07 INFO]: [Skript]  - SkBriggy v1.2.0
[13:42:07 INFO]: [Skript]  - SharpSK v1.6.13
[13:42:07 INFO]: [Skript]  - skUtilities v0.9.2 (https://tim740.github.io/)
[13:42:07 INFO]: [Skript]  - skript-reflect v2.4 (https://github.com/SkriptLang/skript-reflect)
[13:42:07 INFO]: [Skript]  - SkQuery v4.1.10
[13:42:07 INFO]: [Skript]  - SkJade v1.4.3 (https://www.github.com/Ankoki-Dev/SkJade)
[13:42:07 INFO]: [Skript]  - Khoryl v1.1.2
[13:42:07 INFO]: [Skript] Installed dependencies:
[13:42:07 INFO]: [Skript]  - Vault v1.7.3-b131
[13:42:07 INFO]: [Skript]  - WorldGuard v7.0.9+5934e49
[13:42:07 INFO]: [Skript]  - GriefPrevention v16.12.0-d4d7ac4

**Comment by FlameDarkk** [comment]
Also same error in 1.19.4
Changing "circle" to "sphere" just makes skript reload without errors but still no visible effects is playing when using test command provided here : https://github.com/Sashie/skDragonRecode/blob/master/src/main/resources/recode%20tests.sk

**Comment by Vulcano771** [comment]
Hello, please share the full use of that particle

**Comment by Vulcano771** [comment]
the test is kinda outdated

**Comment by FlameDarkk** [comment]
full code is same from the offical example here : https://github.com/Sashie/skDragonRecode/blob/master/src/main/resources/recode%20tests.sk
I also tested this on fresh servers of 1.20.4 and 1.19.4 papermc with only skript and skdragon recode installed.
Folders/Files of test servers I used is here : https://cdn.discordapp.com/attachments/1211796573265207396/1211806292776058910/Desktop.rar?ex=65ef898e&is=65dd148e&hm=514070c259a0fd4834b23504740fde6c21a9d91763f33d2eaebbad0c56260975&

**Comment by Vulcano771** [comment]
I know but the examples dont work xD

**Comment by FlameDarkk** [comment]
Okay... So its not about skript version nor server version. But it seems like some effect names gives error while others work fine.
Sashie is trying to sort it out.
For the note, working(or at least do not give errors in console in the meaning of "working") effect names are ;
sphere, atom, band, blackhole, lasers, meteor

**Comment by Vulcano771** [comment]
Hello, we wont run any things you provide to us on our pc's - especially .jar files or something like that.
Please send us the output of the following Command: (u need Skript-Reflect)
```
command /getCircle:
    trigger:
        broadcast "%circle.getClass().getName()%"
```

**Comment by FlameDarkk** [comment]
Yeah for testing part you don't really need to use the server I used. I just uploaded them in case you guys wanna compare or take a look into.
You can get the same results with the same server/plugin version on windows 10.
the command you gave me is broadcasting something like "sashie.skdragon.particleffect.#6"

**Comment by FlameDarkk** [comment]
I found out that when I local host my server, it gives theese errors.
But if I use hosting service to open my server, it works fine without any error or visible faults.

Further testing needed from anyone other than me to decide if the problem is my local machine or it just doesn't work on local hosted servers with no port forwarding or dns.

**Comment by FlameDarkk** [comment]
On localhosted server
```
command /getCircle:
    trigger:
        broadcast "%circle.getClass().getName()%"
```
gives the following error :
```
[02:30:09 INFO]: Line 16: (dragon.sk)
[02:30:09 INFO]:     Can't understand this expression: circle.getClass().getName()
[02:30:09 INFO]:     Line: broadcast "%circle.getClass().getName()%"
```

**Comment by Sashie** [comment]
So far I have not been able to reproduce this error. I'll keep it open for now in case there is more development on the issue

**Comment by Sashie** [comment]
This was never able to be reproduced outside of the original case, closing this issue for now
